kolibri/core/exams/serializers.py

```diff
@@ -103,6 +115,21 @@ def validate(self, attrs):
                 code=error_constants.UNIQUE,
             )
 
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+        if "question_sources" in data and data["question_sources"]:
+            first_question_source = data["question_sources"][0]
```

Comment:
Seems like this conditionality could just be done based on the `data_model_version` rather than this sniff test?